tests/vanilla/basic.test.ts

```diff
@@ -70,6 +70,13 @@ it('can get the store', async () => {
   expect(store.getState().getState2().value).toBe(1)
 })
 
+it('can get the initial state', () => {
+  const initial = { value: 1 }
+  const store = createStore(() => initial)
+  store.setState({ value: 2 })
+  expect(store.getInitialState()).toBe(initial)
```

Comment:
```suggestion
  const initial = { value: 1 }
  const store = createStore(() => initial)
  
  store.setState({ value: 2 })
  
  expect(store.getInitialState()).toBe(initial)
```

How about adding newline?

Reply:
It's minority, but I usually don't add many lines.